runestone/common/project_template/_templates/plugin_layouts/sphinx_bootstrap/layout.html

```diff
@@ -223,17 +223,28 @@
 
 <script type="text/javascript">
   eBookConfig = {};
-  eBookConfig.host = '{{course_url}}' ? '{{course_url}}' : 'http://127.0.0.1:8000';
-  eBookConfig.app = eBookConfig.host+'/{{appname}}';
+  {% if use_services %}
```

Comment:
@bjones1 -- The only hesitation I have about this is that it conflates using Runestone Ajax services with our move to dynamic pages.  I feel like it would be much more backward compatible to have a separate flag

Reply:
without something a bit more flexible then everyone who runs their own server will need to move to dynamic pages, they won't be able to rebuild any books to work in the old mode once this PR is accepted.